README.md

```diff
@@ -125,32 +92,88 @@ If configured, the action supports setting the Jira issues's status when the Bug
     - mapping [str, str]
     - If defined, map the Bugzilla bug resolution to Jira issue resolution
 
-Example configuration:
+Minimal configuration:
 ```yaml
     whiteboard_tag: example
     contact: example@allizom.com
-    description: example configuration
-    enabled: true
-    module: jbi.actions.default_with_assignee_and_status
+    description: minimal configuration
     parameters:
       jira_project_key: EXMPL
-      status_map:
-        NEW: "In Progress"
-        FIXED: "Closed"
-      resolution_map:
-        FIXED: "Done"
-        DUPLICATE: "Duplicate"
-        WONTFIX: "Won't Do"
+```
+
+Full configuration, that will set assignee, change the Jira issue status and resolution.
+
+```yaml
+- whiteboard_tag: fidefe
+  contact: example@allizom.com
+  description: full configuration
+  enabled: true
+  module: jbi.actions.default
+  parameters:
+    jira_project_key: FIDEFE
+    steps:
+      new:
+      - create_issue
+      - maybe_delete_duplicate
+      - add_link_to_bugzilla
+      - add_link_to_jira
+      - maybe_assign_jira_user
+      - maybe_update_issue_resolution
+      - maybe_update_issue_status
+      existing:
+      - update_issue
+      - add_jira_comments_for_changes
+      - maybe_assign_jira_user
+      - maybe_update_issue_resolution
+      - maybe_update_issue_status
+      comment:
+      - create_comment
+    status_map:
+      ASSIGNED: In Progress
+      FIXED: Closed
+      WONTFIX: Closed
+      DUPLICATE: Closed
+      INVALID: Closed
+      INCOMPLETE: Closed
+      WORKSFORME: Closed
+      REOPENED: In Progress
+    resolution_map:
+      FIXED: Done
+      DUPLICATE: Duplicate
+      WONTFIX: "Won't Do"
 ```
 
 In this case if the bug changes to the NEW status the action will attempt to set the linked Jira
 issue status to "In Progress". If the bug changes to RESOLVED FIXED it will attempt to set the
 linked Jira issue status to "Closed". If the bug changes to a status not listed in `status_map` then
 no change will be made to the Jira issue.
+### Available Steps
+
+- `create_issue`
+- `maybe_delete_duplicate`
+- `add_link_to_bugzilla`
+- `add_link_to_jira`
+- `maybe_assign_jira_user`:
+  It will attempt to assign the Jira issue the same person as the bug is assigned to. This relies on
+  the user using the same email address in both Bugzilla and Jira. If the user does not exist in Jira
+  then the assignee is cleared from the Jira issue. The Jira account that JBI uses requires the "Browse
+  users and groups" global permission in order to set the assignee.
+- `maybe_update_issue_resolution`:
+  If the Bugzilla ticket resolution field is specified in the `resolution_map` parameter, it will set the
+  Jira issue resolution.
+- `maybe_update_issue_status`:
+  If the Bugzilla ticket status field is specified in the `status_map` parameter, it will set the
```

Comment:
These explanations seem like they'd be valuable to include in the function docstrings as well